config/constants/settings/definition.rb

```diff
@@ -163,56 +1154,34 @@ def add(name,
                          writable:,
                          allowed:,
                          env_alias:)
-
         override_value(definition)
-
-        all << definition
+        @all[name] = definition
       end
 
       def define(&)
         instance_exec(&)
       end
```

Comment:
I think this method is not needed anymore and can be removed.

Reply:
:+1: